File: src/butcher/confy.zig
//!:______________________________________________________
//! ᛝ confy  |  Copyright (C) Ivan Mar (sOkam!)  |  MIT  :
//!:______________________________________________________
//! @fileoverview
//!  Confy : Comfortable and Configurable Buildsystem
//!  @todo : Docs here
//_____________________________________________________|
const confy = @This();

//______________________________________
// @section Forward Export zstd tools
//____________________________
const zstd = @import("./lib/zstd.zig");
pub const Name = zstd.Name;


//______________________________________
// @section Forward Export Confy tools
//____________________________
pub const Git      = @import("./confy/git.zig");
pub const Package  = @import("./confy/package.zig");
pub const BuildTrg = @import("./confy/target.zig");

File: butcher/src/deprecated/confy.nim

{.warning:"""
The Nim-based confy toolset is deprecated,
and will be removed as soon as the zig rewrite is completed.
""".}

#:_____________________________________________________
#  confy  |  Copyright (C) Ivan Mar (sOkam!)  |  MIT  :
#:_____________________________________________________
## @fileoverview Cable connector to all modules of the library
import ./confy/types       ; export types
import ./confy/cfg         ; export cfg
import ./confy/dirs        ; export dirs
import ./confy/flags       ; export flags
import ./confy/obj as cObj ; export cObj
import ./confy/builder     ; export builder
import ./confy/tasks       ; export tasks
import ./confy/tools       ; export tools
# @section Forward nstd dependencies to the user's buildsystem code
import nstd/paths   ; export paths
import nstd/strings ; export strings
